### ISSUE TITLE
Dialogue window initial position when using multiple monitors.

### ISSUE THREAD

**Issue by talmaru** [RPTools/maptool#2529]
When using multiple monitors and running map tools on a non-primary monitor some dialogue windows position will revert to the primary monitor.

**To Reproduce**
Open map tools on the primary monitor
Move map tools to the secondary monitor
Drag a token from the repository to the map.

**Expected behavior**
The dialog window should open at the location where the token was dropped onto the map

**MapTool Info**
1.8.4
New windows install

**Desktop (please complete the following information):**
Windows 10 - 19041.867
 

**Additional context**
https://forums.rptools.net/viewtopic.php?f=21&t=29153

**Comment by bubblobill** [comment]
@kwvanderlinde You use multiple monitors. Is this still a thing?

**Comment by kwvanderlinde** [comment]
Yes, definitely so. Although I found it doesn't force it to the primary screen so much as it forces it to the leftmost screen (and probably topmost if using a vertical setup).

The dialog does _try_ to position itself over the drop location. But it stumbles when it tries to fit the dialog to the avaliable screen space, which to it is `(0, 0, primaryMonitor.width, primaryMonitor.height)`, regardless of where MT is located or _where the primary monitor is located_. A better approach IMO would be to either ignore the screens and instead fit it to the parent window, or just centre it like all the other dialogs.
